tiup/tiup-component-cluster-prune.md

```diff
@@ -0,0 +1,25 @@
+---
+title: tiup cluster prune
+---
+
+# tiup cluster prune
+
+When [scaling in the cluster]<!-- (/tiup/tiup-component-cluster-scale-in.md) -->, for some components, TiUP does not immediately stop their services or delete their data. You need to wait for the data scheduling to complete and then manually execute the `tiup cluster prune` command to clean up.
```

Comment:
What this mean? Why comment the url?

Reply:
To avoid ci check, and after all prs are merged, we will add the url back.